webapp/i18n/en.json

```diff
@@ -1,423 +1,2 @@
 {
-  "AppBar.Tooltip": "Toggle Linked Boards",
```

Comment:
I ran the command before committing the changes. Don't know much about `i18n` but these were the changes made when I ran the command.

Reply:
@Rajat-Dabade Should I add it manually?

Reply:
@varunKT001 I tested locally and am not seeing all the data from `en.json` gone. If needed, we can manually run `i18n-extract` for you on this PR but I'd like to run through a few steps before we go that far:

1. confirm you're using the correct Node/npm by running `nvm install` from the `webapp` directory
2. if needed, re-install dependencies (the `i18n-extract` is using `formatjs` under the hood, which is installed as a dep)
3. `cd` to the `webapp` directory and run `npm run i18n-extract`

If you're still having trouble I'd be great if you could share a bit more about any error messages you might be receiving and what environment you're using to contribute on. It might simply be a gap in our documentation. Let us know, thanks!

Reply:
@Pinjasaur @Rajat-Dabade 

I re-installed all the dependencies and run the `npm run i18n-extract` but there is still no data in the `en.json`.

Node version: 16.13.0
NPM version: 8.1.3
Operating system: Windows 10 (21H1)

![image](https://user-images.githubusercontent.com/83509023/195505810-7a6c0878-7d40-47ef-a138-63476ef8530a.png)

There are no errors or warnings thrown when I run the command, so I can't figure out what's the issue here 😕

Reply:
@Pinjasaur @Rajat-Dabade For now, should I manually add the content to the `en.json` file?

Reply:
@varunKT001 I did an `i18n-extract` locally and pushed it here for your PR. Apologies that the command isn't working for you. I'm guessing it's a Windows issue, even though based on your screenshot that looks like Git Bash which I would normally expect to work. It's bizarre there's no error. I wonder if the paths aren't being interpreted right due to filesystem differences. I'll file an issue to track it and see if we can get to the bottom of it.

Reply:
While writing up #4017 I found #3932 which might fix the problems on Windows. Need to investigate further.

Reply:
Okay, got it. Thanks, @Pinjasaur.

Reply:
@Pinjasaur 

I don't know why it's working unexpectedly. It happens very rarely that I face problems like this due to Windows. I'm thinking of using WSL for development purposes. 

BTW thanks for helping 😄